orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/admin/web/QueueAdminController.kt

```diff
@@ -51,6 +55,17 @@ class QueueAdminController(
       )
     )
 
+  @PostMapping(value = ["/killZombies"])
```

Comment:
i think we should support specifying a cutoff time
I am worried that 30m cutoff is too small. This will trip manual judgement stages that are running for a while and wait for execution window/wait stage

Reply:
I bumped the default to 60 minutes, as well as opened it for configuration.